modules/apps/account/account-test/src/testFunctional/tests/AccountRoles.testcase

```diff
@@ -75,4 +75,91 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LPS-144122. This asserts user can view account roles tab when accounts with permission."
+	@priority = "4"
+	test CanViewAccountRolesWithPermissions {
+		task ("Given add a user with permission of view CP and accounts") {
+			JSONUser.addUser(
+				userEmailAddress = "userea1@liferay.com",
+				userFirstName = "userfn1",
+				userLastName = "userln1",
+				userScreenName = "usersn1");
+
+			JSONUser.setFirstPassword(
+				agreeToTermsAndAnswerReminderQuery = "true",
+				requireReset = "false",
+				userEmailAddress = "userea1@liferay.com");
+
+			JSONRole.addRegularRole(roleTitle = "View Account Permission");
+
+			Permissions.definePermissionViaJSONAPI(
+				resourceAction = "VIEW_CONTROL_PANEL",
+				resourceName = "90",
+				roleTitle = "View Account Permission");
+
+			Permissions.definePermissionViaJSONAPI(
+				resourceAction = "ACCESS_IN_CONTROL_PANEL",
+				resourceName = "com_liferay_account_admin_web_internal_portlet_AccountEntriesAdminPortlet",
+				roleTitle = "View Account Permission");
+
+			JSONRole.assignRoleToUser(
+				roleTitle = "View Account Permission",
+				userEmailAddress = "userea1@liferay.com");
+		}
+
+		task ("When add an account with Account Entry Update View and View Account Roles permissions") {
+			JSONAccountEntry.addAccountEntry(
+				accountEntryName = "Account Name",
+				accountEntryType = "Business");
+
+			Accounts.openAccountsAdmin();
+
+			LexiconEntry.gotoEntry(rowEntry = "Account Name");
+
+			Accounts.addOwnedAccountRole(accountRoleTitle = "Account Specific Role");
```

Comment:
The permission for this test should be the one in Role Admin > Account Role, not the owned role in account. Thanks.